assets/src/edit-story/components/checklist/karma/checklist.karma.js

```diff
@@ -346,3 +347,254 @@ describe('Checklist integration', () => {
     expect(fixture.editor.checklist.priorityPanel).toBeDefined();
   });
 });
+
+[true, false].forEach((hasUploadMediaAction) => {
+  describe('Checklist integration - hasUploadMediaAction=false', () => {
+    let fixture;
+
+    beforeEach(async () => {
+      fixture = new Fixture();
+      fixture.setFlags({ enableChecklistCompanion: true });
```

Comment:
🤔  the checklist is enabled by default, is this needed?

Reply:
Oh nice, I can remove this too then!

Reply:
Turns out we can't remove this yet 😕

Reply:
I think it can only be removed once the feature flag code is completely removed from everywhere, including `Experiments.php`. Karma tests don't know that it's been enabled by default in PHP code, obviously :)